### PR TITLE
Design some static pages

### PR DESCRIPTION
**Pull request by tienkane**
## Related Tickets:
- [#26818](https://edu-redmine.sun-asterisk.vn/issues/26818)
- [#26819](https://edu-redmine.sun-asterisk.vn/issues/26819)
- [#26820](https://edu-redmine.sun-asterisk.vn/issues/26820)
- [#26821](https://edu-redmine.sun-asterisk.vn/issues/26821)

## WHAT this PR does?
- Design static pages:
 + Overall category
 + Detail category
 + Contact
 + Request writer
 + Detail post content
 + Tags

## Notes Impacted Areas:
- N/A.

## Attachments:
Screen Shot 2020-08-21 at 11 18 42
Screen Shot 2020-08-21 at 11 19 16
Screen Shot 2020-08-21 at 11 19 56
Screen Shot 2020-08-21 at 11 20 41
Screen Shot 2020-08-21 at 11 18 12

### CODE REVIEW

**Comment by tienkane** [comment]
ready

File: resources/views/components/sidebar_recent_posts.blade.php [hunk 1]
```diff
@@ -0,0 +1,42 @@
+<div class="tab-widget">
+    <ul class="nav nav-tabs">
+        <li class="active"><a href="#recent" data-toggle="tab">{{ trans('app.recent') }}</a></li>
+        <li><a href="#popular" data-toggle="tab">{{ trans('app.popular') }}</a></li>
+    </ul>
+    <div class="tab-content">
+        <div class="tab-pane fade in active" id="recent">
+            {{-- Data recent posts --}}
+            @for ($i = 1; $i < 5; $i++)
```
Comment (huyenttm-0901) [inline resources/views/components/sidebar_recent_posts.blade.php:9]: 5 là số gì v em? cho vào config nhé

File: resources/views/components/sidebar_recent_posts.blade.php [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+                <div class="media latest_post">
+                    <a class="media-left" href="javascript:void(0)">
+                        <img src="{{ asset("/bower_components/osru-template-assets/assets/images/100x70-$i.jpg") }}" class="media-object" alt="">
+                    </a>
+                    <div class="media-body">
+                        <h6 class="media-heading"><a href="javascript:void(0)">The <em>Best Street-Style</em> Pics Copenhagen</a></h6>
+                        <div class="entry-meta">
+                            <span class="entry-date"><i class="fa fa-calendar-o" aria-hidden="true"></i><time datetime="2018-01-21T19:00">Jan 21, 2018</time> </span>
```
Comment (huyenttm-0901) [inline resources/views/components/sidebar_recent_posts.blade.php:17]: data example em comment hết lại đi

File: resources/views/components/sidebar_recent_posts.blade.php [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+                        </div>
+                    </div>
+                </div>
+            @endfor
+            {{-- End data recent posts --}}
+        </div>
+        <div class="tab-pane fade" id="popular">
+            {{-- Data popular posts --}}
+            @for ($i = 1; $i < 5; $i++)
```
Comment (huyenttm-0901) [inline resources/views/components/sidebar_recent_posts.blade.php:26]: tương tụ

File: resources/views/components/sidebar_tags.blade.php [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+    <a href="javascript:void(0)">HTML</a>
+    <a href="javascript:void(0)">CSS</a>
+    <a href="javascript:void(0)">PHP</a>
+    <a href="javascript:void(0)">Fashion</a>
+    <a href="javascript:void(0)">News</a>
+    <a href="javascript:void(0)">Blog</a>
+    <a href="javascript:void(0)">Travel</a>
+    <a href="javascript:void(0)">Politics</a>
+    <a href="javascript:void(0)">Economy</a>
+    <a href="javascript:void(0)">Javascript</a>
```
Comment (huyenttm-0901) [inline resources/views/components/sidebar_tags.blade.php:15]: data này ở đâu ra v em?

File: resources/views/frontend/contact.blade.php [hunk 5]
```diff
@@ -0,0 +1,80 @@
...
+        <div class="container">
+            <div class="row">
+                <div class="col-sm-6">
+                    <div class="contact-info">
+                        <div class="contact-address">
+                            <div class="contact-text">
+                                <h3>{{ trans('app.contact') }}</h3>
+                                <p>Contrary to popular belief, Lorem Ipsum is not simply random text. It has roots in a piece of classical Latin literature from 45 BC, making it over 2000 years old. Richard McClintock, a Latin professor at Hampden-Sydney College in Virginia.</p>
+                            </div>
+                            <div class="address-info">
```
Comment (huyenttm-0901) [inline resources/views/frontend/contact.blade.php:29]: tương tự

**Review by huyenttm-0901 — COMMENTED** [review]
sửa hết i18n, các phần data example và config

**Comment by tienkane** [comment]
ready

**Pull request merged**